src/WebJobs.Script/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -40,6 +44,21 @@ public static bool IsHttpTriggerFunction(this FunctionMetadata metadata)
             return metadata.InputBindings.Any(b => string.Equals(HttpTriggerKey, b.Type, StringComparison.OrdinalIgnoreCase));
         }
 
+        public static bool IsLegacyBlobTriggerFunction(this FunctionMetadata metadata)
```

Comment:
Does this helper work when there is no function.json - i.e. in worker indexing situations?

Reply:
It should work if the worker returns all required function metadata. 

Getting functions metadata - host or worker indexing: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Host/ScriptHost.cs#L279

Validating functions: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Host/ScriptHost.cs#L314